multimodal/src/autogluon/multimodal/utils/metric.py

```diff
@@ -178,6 +180,171 @@ def compute_score(
         return metric._sign * metric(metric_data[Y_TRUE], metric_data[Y_PRED])
 
 
+class RankingMetrics:
+    def __init__(
+        self,
+        pred: Dict[str, Dict],
+        target: Dict[str, Dict],
+        is_higher_better=True,
```

Comment:
To be consistent with our previous code, here we can call it "min_max_mode". I think "is_higher_better" is easier to understand though.

Reply:
min_max_mode is used for metrics, here is_higher_better is used to indicate how we should rank the list. They are essentially different